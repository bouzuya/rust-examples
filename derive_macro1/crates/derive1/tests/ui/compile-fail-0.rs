fn main() {
    #[derive(derive1::VariantsFn)]
    enum E {
        #[rename]
        A,
        B,
        C,
    }
}
