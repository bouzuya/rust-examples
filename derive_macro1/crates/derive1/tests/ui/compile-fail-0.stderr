error: expected `#[rename = "name"]` attribute
 --> tests/ui/compile-fail-0.rs:4:9
  |
4 |         #[rename]
  |         ^
